python/ray/data/dataset.py

```diff
@@ -367,7 +370,8 @@ def flat_map(
             fn: The function to apply to each record, or a class type
                 that can be instantiated to create such a callable.
             compute: The compute strategy, either "tasks" (default) to use Ray
-                tasks, or "actors" to use an autoscaling Ray actor pool.
+                tasks, "actors[n]" for an actor pool of size `n`, or "actors" to
```

Comment:
What about `actors[0]`? Could you add a test case for it?

Reply:
Added